src/stories/nve-input/NveInputDoc.mdx

```diff
@@ -0,0 +1,114 @@
+import './../css/stories.css';
+
+import {
+  Canvas,
+  Story,
+  Meta,
+  Title,
+  Subtitle,
+  Description,
+  Source,
+  Primary,
+  Controls,
+  Stories,
+} from '@storybook/blocks';
+
+import * as NveInputStories from './NveInput.stories';
+
+<Meta isTemplate />
+
+<Title />
+<Description />
+<br />
+<br />
+
+## Størrelser
+
+<div class="container">
+  <span class="elem">
+    <Canvas of={NveInputStories.Small} />
+  </span>
+  <span class="elem">
+    <Canvas of={NveInputStories.Medium} />
+  </span>
+  <span class="elem">
+    <Canvas of={NveInputStories.Large} />
+  </span>
+</div>
+
+## Filled
+
+<div class="container">
+  <span class="elem">
+    <Canvas of={NveInputStories.Filledone} />
+  </span>
+  <span class="elem">
+    <Canvas of={NveInputStories.Filledtwo} />
+  </span>
+  <span class="elem">
+    <Canvas of={NveInputStories.Filledthree} />
+  </span>
+</div>
+
+## Aktiv/Deaktivert
+
+<div class="container">
+  <span class="elem">
+    <Canvas of={NveInputStories.Enabled} />
+  </span>
+  <span class="elem">
+    <Canvas of={NveInputStories.Disabled} />
+  </span>
+</div>
+
+## Obligatorisk/valideringsfeil
+
+<div class="container">
+  <span class="elem">
```

Comment:
Her kan du kanskje ha tre varianter:
- En hvor du ikke spesifiserer requiredLabel, da skal du få "*Obligatorisk" som standard
- En hvor du ikke spesifiserer requiredLabel, f.eks. på et annet språk enn norsk
- Et med min- og maks, som du har, men ta gjerne med help-text som et hint om hva som skal skrives inn i tekstfeltet

Reply:
ok nå, La inn logikk for det i storybook